Doc/library/datetime.rst

```diff
@@ -14,40 +14,59 @@
 
 .. XXX what order should the types be discussed in?
 
-The :mod:`datetime` module supplies classes for manipulating dates and times in
-both simple and complex ways.  While date and time arithmetic is supported, the
-focus of the implementation is on efficient attribute extraction for output
-formatting and manipulation. For related functionality, see also the
-:mod:`time` and :mod:`calendar` modules.
+The :mod:`datetime` module supplies classes for manipulating dates and times.
 
-There are two kinds of date and time objects: "naive" and "aware".
+While date and time arithmetic is supported, the focus of the implementation is
+on efficient attribute extraction for output formatting and manipulation.
 
-An aware object has sufficient knowledge of applicable algorithmic and
+.. seealso::
+
+   Module :mod:`calendar`
+      General calendar related functions.
+
+   Module :mod:`time`
+      Time access and conversions.
+
+   Package `dateutil <https://dateutil.readthedocs.io/en/stable/>`_
+      Third-party library with expanded time zone and parsing support.
+
+.. _datetime-naive-aware:
+
+Naive vs Aware Objects
+----------------------
+
+There are two kinds of date and time objects: "naive" and "aware."
+
+An **aware** object has sufficient knowledge of applicable algorithmic and
```

Comment:
With sufficient knowledge of applicable algorithmic and political time adjustments, such as time zone and daylight saving time information, an **aware** object can locate itself relative to other aware objects. An aware object represents a specific moment in time that is not open to interpretation.